File: DMOPC/2016/Contest_1/P0.c
#include <stdio.h>

int main(){
    int w,c;
    scanf("%d\n%d",&w,&c);
    char* s = "very";
    if(w==3&&c>94)s="absolutely";
    else if (w==1&&c<51)s="fairly";
    printf("C.C. is %s satisfied with her pizza.",s);
    return 0;
}

File: DMOPC/2016/Contest_1/P0.rs
use std::io;
macro_rules! parseln {
    ($($t: ty),+) => ({
        let mut a_str = String::new();
        io::stdin().read_line(&mut a_str).expect("read error");
        let mut a_iter = a_str.split_whitespace();
        (
            $(
            a_iter.next().unwrap().parse::<$t>().expect("parse error"),
            )+
        )
    })
}

fn main() {
    let w = parseln!(u32).0;
    let c = parseln!(u32).0;
    let mut s = "very";
    if w == 3 && c > 94 { s = "absolutely"; }
    else if w == 1 && c < 51 { s = "fairly"; }
    println!("C.C. is {} satisfied with her pizza.", s);
}
